README.md

```diff
@@ -1,80 +1,104 @@
 
-# Article IV
-## Membership Requirements and Limitations
+### Membership Requirements and Limitations
 
 Membership to this organization requires that the student be enrolled at Clemson University full time or part time. Both undergraduate and graduate level students will be allowed to join. Members will be required to attend a minimum of 2 meetings per semester. No requirements will be established regarding major or qualifications. Membership is open to all students of Clemson University who meet the requirements listed previously.
 
+## Article V
 
-# Article V
-## Officers (Titles, Term of Office, How and When Elected)
+### Officers (Titles, Term of Office, How and When Elected)
 
-Each officer position with the exception of System Architect serves until the Spring semester election following their election. Nominations for each generally elected position, that is all positions other than System Architect, can be submitted by any member beginning one month before an election and ending at the election of the particular officer. At the last meeting of the Spring semester or the last meeting of the Fall semester, the generally elected positions that are open, that is all positions other than System Architect that are open, will be filled by a majority vote of the nominees by active members. The position of System Architect will be filled by a unanimous vote of the current officers whenever the position is open.
+Each officer position serves until the Spring semester election following their election. Nominations for each generally elected position can be submitted by any member beginning one month before an election and ending at the election of the particular officer. At the last meeting of the Spring semester or the last meeting of the Fall semester, the generally elected positions that are open will be filled by a majority vote of the nominees by active members.
 
 Officer positions are as follows:
+
 * President
 * Vice President
-* System Architect
 * Treasurer
 * Secretary
-* Webmaster
+* Competition and Events Manager
+* Training Manager
+
+Should an officer be unable to fulfill their duties due to health issues, they are relieved of their position, and their position is to be filled at the next meeting by a snap election.
+
+## Article VI
+
+### Appointed Positions (Titles, Term of Office, How Selected)
+
+Each appointed position, with the exception of Lab Administrator, serves at-will of the board of Officers, is appointed by 2/3 majority of Officers' approval, and is relieved via graduation or 2/3 majority of sitting Officers' approval. The position of Lab Administrator will be filled by a unanimous vote of the current officers and advisors, relieved only via graduation or unanimous vote of current officers and advisors.
+
+Appointed positions are as follows:
+
+* Lab Administrator
+* Website Administrator
+* Social Event Manager
+* CyberRange Project Manager
+* Weekly Meeting Manager
+* Blue Team Trainer
+* Red Team Trainer
```

Comment:
This is a large number of appointed positions for an organization the the size of CUCyber.  The way I see it, there should be something addressing the following cases:

1. There are insufficient people for each office to be held by separate individuals.
2. There are insufficient people for each office to be held by _a_ person.

Reply:
I think appointed positions were meant to be overlapping functional roles, as opposed to distinct individuals. Not sure on that though

Reply:
That's not at all clear from this text.  Especially since Officers must be distinct natural persons.

Reply:
That's why these aren't officers, and are under their own article

Reply:
Is this: 

> There are insufficient people for each office to be held by separate individuals

Not addressed by 

https://github.com/CUCyber/constitution/pull/5#discussion_r338284754

specifically,

> Appointed positions may be filled by any active member who is not an officer, and a member may hold more than one appointed position.

As for

> There are insufficient people for each office to be held by a person.
an solution that involves absolving the officers of the necessity of appointing appointee positions based on officer discretion could be abused or misinterpreted. In other words, if it is left to the discretion of the officers to define "for each office to be held by a person." they may use this clause to neglect to appoint positions that it's for some reason inconvenient to appoint. 

One possible solution is to define a hard-minimum number of officers above which all positions can be reasonably expected to be filled.

Reply:
One way around this would be to leave the appointed positions other than "Lab Administrator" optional, with a clause such as:

> The president with consent of 2/3rds of the other officers may appoint such positions as he or she deems necessary and properer.

This allows the extra positions to silently disappear/appear without constitutional amendment as needed.

Reply:
This will be more easily achieved by separating out the conversion of the lab admin into a separate amendment.

Should system architect/lab admin be its own section or Article? Wondering since the stipulations that apply to it are so different, and the access requirements are much greater.